README.md

```diff
@@ -0,0 +1,43 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Hello Microverse repository contains html and css.
+
+![screenshot](./hello-microverse.png)
+
+Hello Microverse project.
```

Comment:
- [ ] [OPTIONAL] For the next projects you should add a better description, just try to explain what you did, what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description from your README file for the next time 😉).